packages/manager/apps/telecom/src/app/telecom/telephony/line/assist/rma/rma.html

```diff
@@ -117,50 +119,102 @@ <h1 data-translate="telephony_line_assist_rma_title"></h1>
+                    target="_top"
+                ></a>
+                <!-- CANCEL RMA -->
+                <button
+                    type="button"
+                    class="btn btn-danger"
+                    data-ng-disabled="rma.cancellable === 'true'"
+                    data-translate="telephony_line_assist_rma_cancel"
+                    data-ng-click="$ctrl.generateCancelRmaModal(rma)"
+                ></button>
+            </dl>
 
-            <!-- DOWNLOAD PDF -->
-            <a
-                class="btn btn-default mr-2 float-right"
-                data-ng-href="{{ RmaCtrl.getPdfUrl(rma) }}"
-                data-translate="telephony_line_assist_rma_pdf_download"
-                target="_top"
-            ></a>
+            <dl class="row" data-ng-if="rma.type === 'unknown'">
+                <!-- KEEPING THE LINE WITHOUT EQUIPMENT -->
+                <button
+                    class="btn btn-default"
+                    data-ng-click="$ctrl.generateKeepingLineModal(rma)"
+                    data-translate="telephony_line_assist_rma_keeping_line"
+                ></button>
+                <!-- REPORTING EQUIPEMENT OUT OF ORDER -->
+                <button
+                    type="button"
+                    class="btn btn-default"
+                    data-ng-click="$ctrl.generateEquipmentOutOfOrderModal(rma)"
+                    data-translate="telephony_line_assist_rma_equipment_out_of_order"
+                ></button>
+                <!-- TERMINATE THE LINE -->
+                <button
+                    type="button"
+                    class="btn btn-danger"
+                    data-translate="telephony_line_assist_rma_terminate_line"
+                    data-ng-click="$ctrl.generateTerminateLineModal(rma)"
+                ></button>
+            </dl>
+        </div>
+    </div>
+
+    <div data-ng-if="$ctrl.isOpenModal">
+        <div class="modal d-block">
+            <div
+                class="modal-dialog modal-dialog-centered modal-dialog-scrollable"
+            >
+                <div class="modal-content">
+                    <form>
+                        <oui-modal
+                            heading="{{:: $ctrl.modal.title | translate }}"
+                            primary-action="$ctrl.executeAction($ctrl.modal.rma)"
+                            primary-label="{{ :: 'telephony_line_assist_rma_modal_yes' | translate }}"
+                            primary-disabled="$ctrl.actionDoing"
+                            secondary-action="$ctrl.resetModalInfo()"
+                            secondary-label="{{ :: 'telephony_line_assist_rma_modal_no' | translate }}"
+                            on-dismiss="$ctrl.resetModalInfo()"
+                        >
+                            <p
+                                data-translate="{{:: $ctrl.modal.content }}"
+                                data-translate-values="{ 'breakLine': '</br>' }"
```

Comment:
**nitpick(non-blocking):** Might be interesting to split the actual translation into several chunk.

Reply:
I fixed up it.